CHANGELOG.md

```diff
@@ -43,6 +43,9 @@
 - [Handle syntax errors in custom-defined visualizations][#1341]. The IDE is now
   able to run properly, even if some of the custom-defined visualisations inside
   a project contain syntax errors.
+- [Fix pasting multiline code in nodes][1348]. Since nodes contain only a single
+  line, the first copied line will be inserted and all additional lines will be
+  ignored.
```

Comment:
The changelog description should be more user-facing. This is a technical description about what happens under the hood. Instead, I'd go in the direction of "[Fixed issues with pasting multi-line text into single-line text fields.][...]" But also, please confirm with any native speaker (Joe, Ara), that this sentence sounds good (there is also a small grammar problem in yours if I'm correct - the "only" should be put at the end of the first part of the sentence). It's important that our public communication is 100% correct English.

Reply:
I would suggest that a native speaker goes over all the texts once before the release. Does that make sense?

Reply:
Not really, the README is a publicly available document on our GitHub. People will be reading that before release.

Reply:
Okay. I updated the text anyway, so the "only" is not there anymore.